### PR TITLE
Refactoring and rename functions

### PR DESCRIPTION
**Pull request by shcherbyakdev**

### CODE REVIEW

File: handler.js [hunk 1]
```diff
@@ -1,33 +1,15 @@
...
 
 module.exports.getVenues = (event, context, callback) => {
 
 try{
- const inputs = inputsData(JSON.parse(event.body));
-
-
-foursquareApiData(inputs).then(function(data){
+ const inputs = helpers.processInputsData(JSON.parse(event.body));
+ helpers.getFoursquareApiData(inputs).then(function(data){
```
Comment (KnupMan) [inline handler.js:9]: Better to use `return` statement for this Promise to be sure, that lambda will wait for the response

Comment (vladholubiev) [reply handler.js:9]: It will wait anyway, it doesn't have `context.dontWaitForEmptyEventLoop`

File: handler.js [hunk 2]
```diff
@@ -1,33 +1,15 @@
...
 
-const columnNames = ["Name", "City", "Street", "Latitude", "Longitude"];
+const response = helpers.responseCsvVenue(data);
```
Comment (KnupMan) [inline handler.js]: Better name for this function will be something like `getResponseCSVVenue`, name should contain an action

Comment (vladholubiev) [inline handler.js]: `responseCsvVenue ` - this is not a verb as well

File: helpers/helperFunctions.js [hunk 3]
```diff
@@ -22,4 +29,70 @@ exports.getFieldsFormApi = function (bodyStr) {
...
+
+
+        const json2csvParser = new Json2csvParser({ columnNames,quote: ' ' });
+        const csv = json2csvParser.parse(data);
+        
+          const responseCsv = {
+            statusCode: 200,
+            headers: {'Content-Type' : 'text/csv; charset=utf-8'},
+            body: JSON.stringify({
+            message: csv
```
Comment (vladholubiev) [inline helpers/helperFunctions.js:92]: next PR - to change this

https://github.com/shcherbyakdev/find-places/issues/1

**Review by vladholubiev — CHANGES_REQUESTED** [review]
Please rename `helpers.responseCsvVenue(data)` to verb as well

File: helpers/helperFunctions.js [hunk 4]
```diff
@@ -22,4 +29,70 @@ exports.getFieldsFormApi = function (bodyStr) {
...
+        const columnNames = ["Name", "City", "Street", "Latitude", "Longitude"];
+
+
+        const json2csvParser = new Json2csvParser({ columnNames,quote: ' ' });
+        const csv = json2csvParser.parse(data);
+        
+          const responseCsv = {
+            statusCode: 200,
+            headers: {'Content-Type' : 'text/csv; charset=utf-8'},
+            body: JSON.stringify({
```
Comment (KnupMan) [inline helpers/helperFunctions.js:91]: Should be w/o stringify, I guess

Comment (vladholubiev) [reply helpers/helperFunctions.js:91]: This is next PR, #1

**Review by vladholubiev — APPROVED** [review]

**Pull request merged**